pkg/etcd-backup-operator/controller/sync.go

```diff
@@ -14,6 +14,7 @@ const (
 	//
 	// 5ms, 10ms, 20ms, 40ms, 80ms, 160ms, 320ms, 640ms, 1.3s, 2.6s, 5.1s, 10.2s, 20.4s, 41s, 82s
 	maxRetries = 15
+	s3         = "s3"
```

Comment:
there is already a StorageType for s3 defined

Reply:
BackupStorageTypeS3               = "S3"

Reply:
do you mean here? https://github.com/coreos/etcd-operator/blob/6a8e08ffc11cfc4dcd4f178ad1664cde16971bcb/pkg/apis/etcd/v1beta2/backup.go#L24